submissions/Vitaminvp/HTML&CSSPractice/style.css

```diff
@@ -0,0 +1,237 @@
+body {
+    font-family: "Arial", sans-serif;
+    background-color: #fff;
+    overflow-x: hidden;
```

Comment:
Also, as I know, it's not a good practice to use `overflow: hidden;` for `body` element. Try to find all the elements in your code which could add unnecessarily scroll on X-axis in browsers.

Reply:
Please, resolve comments, where you have already done fixes.